bindings/c/test/mako/mako.c

```diff
@@ -870,14 +870,19 @@ int worker_process_main(mako_args_t *args, int worker_id, mako_shmhdr_t *shm) {
 
   /* enable knobs if specified */
   if (args->knobs[0] != '\0') {
-    if (args->verbose >= VERBOSE_DEBUG) {
-      printf("DEBUG: Setting client konbs: %s\n", args->knobs);
-    }
-    err = fdb_network_set_option(FDB_NET_OPTION_KNOB, (uint8_t *)args->knobs,
-                                 strlen(args->knobs));
-    if (err) {
-      fprintf(stderr, "ERROR: fdb_network_set_option: %s\n",
-              fdb_get_error(err));
+    char delim[] = ", ";
```

Comment:
Not sure why this change is part of the patch, but if TCP_QUICKACK knob is supposed to be tuned via this I think you may have to update `fdb.options` file based on quick look at the definition of `fdb_network_set_option`.

Reply:
This change was needed to pass multiple knobs to mako benchmark.  I agree this is a general enhancement, not tied to TCP_QUICKACK, but this was the first time I needed this.